client/src/main/java/com/tencent/rss/client/impl/ShuffleWriteClientImpl.java

```diff
@@ -284,9 +284,13 @@ public void reportShuffleResult(
           break;
         }
       } catch (Exception e) {
```

Comment:
catch (InterruptedException ie) {

} catch(Exception e) {

}

Reply:
Can't catch InterruptedException, because it is never thrown in try block.

Reply:
can catch.